src/styles/blocks/header.scss

```diff
@@ -0,0 +1,104 @@
+body,
+h1,
+ul,
+li,
+p,
+a {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The styles for the global tags (body, h1, ul, li, p, a) should not be in the header.scss file. These are global styles and they should be in a separate file (like base.scss or reset.scss). This is a code organization issue.